app/data/events.json

```diff
@@ -1,90 +1,104 @@
 [
 	{
-		"date": "2023-06-29",
+		"date": "2023-06-29T18:30:00.000-05:00",
```

Comment:
We're in UTC-4 right now!  We'll have to come up with some sort of process to make sure that we get the timezones right here, so that our calendar feed isn't off by an hour when the timezones inevitably change… it would be nice to be able to do that with code, but I don't have a good answer off the top of my head.

Reply:
We also have to correct the durations of all events; what would be your approach to that? Is there a collection of all the meetups I can access to update this file easily?